### DISCUSSION TITLE
Could I get the error info from CMakeApp (RDT-624)

### DISCUSSION THREAD

**Discussion by feizi** [espressif/idf-build-apps#101] [Q&A] [answered]
Is there any way to get error messages when I compile app by using CMakeApp.

**Comment by hfudev** [comment]
Hi @feizi, thank you for the question.

when you're building it by (here's a simple example, assume running it under $IDF_PATH)

```python
CMakeApp('examples/get-started/hello_world', 'esp32').build()
```

then the cmake output should be printed to the stdout by default.

Are you requesting something else?

**Comment by feizi** [comment]
I wonder if I can get the parsed error messages.
For example:
```
{message: "expected ';' before '}' token", position:"main.c:7:29:"}
...
```

**Comment by hfudev** [comment]
Okay, I see. Thank you. I'll create a feature request and move the current issue to discussion.

**Reply by hfudev** [reply, marked as answer]
noted here: https://github.com/espressif/idf-build-apps/issues/102
